run.bat

```diff
@@ -0,0 +1,5 @@
+gcc -o selfie selfie.c
+move selfie.mips out
+selfie.exe < selfie.c
+move out selfie.mips
+selfie.exe -m 1 selfie.mips < selfie.c
```

Comment:
is this our last pull request? it should be from akuche (29th of October at 11:57 pm) and it's in the closed pull requests list